packages/studio/src/utils/PageDataValidator.ts

```diff
@@ -0,0 +1,87 @@
+export interface ValidationResult {
+  valid: boolean;
+  errorMessages: string;
```

Comment:
I think we should have this be an array of strings. Then, the consumer of this class can choose how it wants to format that. I don't think we should prescribe formatting in this class.